analysis/01_trigger_threshold_tables.qmd

```diff
@@ -158,25 +177,74 @@ gt_primera <- df_primera_thresholds %>%
       columns = c("Guatemala"), 
       rows = 3
     )) %>% 
-  # tab_footnote(
-  #   footnote= gt_legend
-  # ) %>% 
-  tab_footnote(footnote = html("Threshold calculations are made based on analysis of historical forecast data (1981-2022) to approximate a 1 in 4 year return period drought levels for rainfall over the entire season. Thresholds are calculated per country, leadtime, and forecast data source to minimize potential biases. Where possible national forecasts were used for this analysis and monitoring. Where no national forecasts were readily available, ECMWF seasonal forecasts/historical forecasts were used. <br><br><b> Note:</b> The national forecast provided by INSUVIMEH in Guatemala does not provide a forecast estimate for the month of publication, therefore for the final month of primera monitoring we use ECMWF data for all 4 countries")) %>% 
+  tab_footnote(footnote = html(txt_footnote)) %>% 
   tab_header(title = "Primera (MJJA 2024) Rainfall (mm) Monitoring Thresholds",
-             subtitle = gt_legend
+             subtitle = gt_mars_legend
              )
 
+gt_cds_legend <- data.frame(
+  `Forecast Data Source` = "Forecast Data Source",
+  ecmwf = "ECMWF (CDS)" ,
+    insuv= "INSIVUMEH"
+
+) %>% 
+  gt(rowname_col = "Forecast Data Source") %>%  
+  data_color(columns = "ecmwf", direction = "column",color=ecmwf_color) %>% 
+  data_color(columns = "insuv", direction = "column",color=insuv_color) %>% 
+  tab_options(
+    column_labels.hidden = T
+  ) %>% 
+  as_raw_html()
+
+gt_primera_cds_set <- df_primera_table$CDS_SET %>% 
+  gt() %>% 
+  fmt_number(decimals = 0) %>% 
+  cols_label(
+    pub_month_lt = html("Publication Month<br>(leadtime)")
+  ) %>% 
+  data_color(columns = ecmwf_cols, direction = "column",color=ecmwf_color) %>% 
+   tab_style(
+    style = cell_fill(color = insuv_color),
+    locations = cells_body(
+      columns = c("Guatemala"), 
+      rows = 1:2
+    )) %>% 
+   tab_style(
+    style = cell_fill(color = ecmwf_color),
+    locations = cells_body(
+      columns = c("Guatemala"), 
+      rows = 3
+    )) %>% 
+  tab_footnote(footnote = html(txt_footnote)) %>% 
+  tab_header(title = "Primera (MJJA 2024) Rainfall (mm) Monitoring Thresholds",
+             subtitle = gt_cds_legend
+             )
 
 
 
-gt_primera
+
+gt_primera_cds_set
+gt_primera_mars_set
 ```
 
+
+
+
 ## Postera
 
 ```{r}
```

Comment:
To avoid confusion, I would put the order of MARS and CDs the same for primera and postera.

Reply:
it looks to me like they are in the same order -- i.e rows and columns in the output tables? maybe you mean something else.

Reply:
ah realized what you meant and fixed it upstream [650722f](https://github.com/OCHA-DAP/ds-aa-lac-dry-corridor/pull/16/commits/650722f96e597a468de2d261d0313b5dc7ec882a)